extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -923,6 +925,9 @@ export class AmpA4A extends AMP.BaseElement {
         return;
       }
       return this.mutateElement(() => {
+        // Fire a event so that 3rd parties know an ad has changed.
+        this.maybeTriggerAnalyticsEvent_('adRefresh');
```

Comment:
I also wasn't sure which variables should be sent with this new event, so I just used the existing a4a vars.